app/controllers/concerns/keywords.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Keywords
+  extend ActiveSupport::Concern
+
+  private
+
+  def search_keyword(user)
+    user.keywords.where("keyword ILIKE '%#{params[:search]}%'").order(keyword: :asc).page(params[:page])
```

Comment:
<img width="863" alt="Screen Shot 2020-11-02 at 17 01 18" src="https://user-images.githubusercontent.com/1896814/97855265-16e87900-1d2d-11eb-9700-bfcb1d5f92b1.png">
This implementation isn't safe. You should use [array condition](https://guides.rubyonrails.org/v3.2.2/active_record_querying.html#array-conditions) instead

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/19